tests/test_copies.py

```diff
@@ -368,27 +471,158 @@ def test_summary_metrics():
         np.testing.assert_almost_equal(metrics[name], expected_results[name], decimal=6)
 
 
-def test_ClassifierCopy_copy_classifier():
-    # Original classifier
-    train_data = pd.DataFrame(
-        {"x": [0, 1, 0, 2, 1], "y": [1, 0, 2, 0, 1], "label": [0, 0, 1, 1, 1]},
-        columns=["x", "y", "label"],
+def test_multivariable_density_comparison(example_presc_datasets):
+    num_figures_before = plt.gcf().number
+    multivariable_density_comparison(
+        [example_presc_datasets[0].df[:-2], example_presc_datasets[1].df[:-2]],
+        feature1="feature1",
+        feature2="feature2",
+        label_col="letter",
+        titles=["Classifier 1", "Classifier 2"],
     )
-    original_classifier = SVC(kernel="linear", random_state=42)
-    original_classifier.fit(train_data[["x", "y"]], train_data["label"])
+    num_figures_after = plt.gcf().number
+    # Checks that the figure was plotted
+    assert num_figures_after == num_figures_before + 1
 
-    # Copy classifier
+
+def test_keep_top_classes(example_presc_datasets):
+    min_num_samples = 3
+    classes_to_keep = ["b", "c"]
+
+    dataset_majority_classes = keep_top_classes(
+        example_presc_datasets[0], min_num_samples=min_num_samples
+    )
+    dataset_specified_classes = keep_top_classes(
+        example_presc_datasets[0],
+        min_num_samples=min_num_samples,
+        classes_to_keep=classes_to_keep,
+    )
+
+    assert dataset_majority_classes.labels.isin(["a", "b"]).all()
+    assert dataset_specified_classes.labels.isin(["b", "c"]).all()
+
+
+def test_SyntheticDataStreamer(
+    trained_original_classifier, instantiated_classifier_copies
+):
+    # Define queue
+    data_stream = Queue(maxsize=4)
+
+    # Instantiate and start data streamer
+    classifier_copy = instantiated_classifier_copies["uniform_copy"]
+    data_streamer = SyntheticDataStreamer(classifier_copy, data_stream, verbose=True)
+    data_streamer.deamon = True
```

Comment:
I must have made a copying mistake. Thanks! I've fixed it. 

I think it is necessary. At first I didn't add that, and then if the test failed it blocked my computer. After adding this, when the test ends the threads also ends, and it may give an error but it doesn't become any major problem.